apps/graphql/src/apps/location/dataloaders/Locations.js

```diff
@@ -17,12 +17,34 @@ export type LocationInput =
       +code: string,
     |};
 
+function sanitizeLocationType({ type }: ApiLocation) {
+  if (type) {
+    return type.replace(/_(\w)/g, m => m[1].toUpperCase());
```

Comment:
could you add a comment as to why this is necessary?

Reply:
done

Reply:
Ok, now that I know why we does this, it seems unnecessary to me. 

```
BUS_STATION: {
      value: 'bus_station',
},
```

will resolve this to the enum anyway.

Reply:
🚀